template/web/app/paymaster-bundler/constants.ts

```diff
@@ -0,0 +1,29 @@
+export const rpcUrl =
+  'https://api.developer.coinbase.com/rpc/v1/base-sepolia/Nz9nSdr8uIY7HhIi2zRrJFgISslFF2In';
+export const paymasterUrl =
+  'https://api.developer.coinbase.com/rpc/v1/base-sepolia/Nz9nSdr8uIY7HhIi2zRrJFgISslFF2In';
```

Comment:
Does this need to be API key from Pimlico?

Reply:
done in new commit, we're using the Cloud Paymaster here